systemtests/service_test.go

```diff
@@ -900,6 +896,7 @@ func (s *systemtestSuite) createServices(c *C, numServices int, tenant string, s
 		services = append(services, service)
 	}
 
+	/* FIXME:
```

Comment:
shouldn't check in commented code: either fix it, or remove it.
what code is is supposed to be fixing? what is broken if this is not fixed?